app/Transformers/JsonApiMemberCustomFieldsDataArrayTransformer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Transformers;
+
+use League\Fractal\TransformerAbstract;
+use App\ProjectCustomField;
+use App\MemberCustomFieldData;
+use App\Transformers\JsonApiProjectCustomFieldTransformer;
+
+class JsonApiMemberCustomFieldsDataArrayTransformer extends TransformerAbstract
+{
+    protected $availableIncludes = [
+        'project-custom-field'
+    ];
+
+    public function transform($value)
+    {
+        $payload = unserialize($value->data);
+
+        return [
+            'id' => $value->id,
+            'data' => $payload->get(),
+            'created_at' => $value->created_at,
+            'updated_at' => $value->updated_at,
+        ];
+    }
+
+    public function includeProjectCustomField($value)
+    {
+        $projectCustomField = ProjectCustomField::find($value->project_custom_field_id);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\App\ProjectCustomField' in method 'includeProjectCustomField'.](https://www.codacy.com/app/ym-huang0808/VMS/file/2452055321/issues/source?bid=2972826&fileBranchId=3248926#l30)